js/apps/events-page/constants.js

```diff
@@ -27,7 +27,11 @@
 			eventDetailsPartial: '/_js/apps/events-page/partials/eventDetails.html',
 			eventRegistrationPartial: '/_js/apps/events-page/partials/eventRegistration.html'
 		},
-		requestChunkSize: 10
+		requestChunkSize: 10,
+		messages: {
```

Comment:
i know this is kinda a pain, but **should** this maybe be reoganzied a bit

```javascript
ageDisclaimer: {
  message: '...',
  groups: [...]
}
```

Reply:
Done!